src/Features/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,7 @@
-
+*REMOVED*static Microsoft.CodeAnalysis.Completion.CompletionItem.Create(string displayText, string filterText = null, string sortText = null, System.Collections.Immutable.ImmutableDictionary<string, string> properties = null, System.Collections.Immutable.ImmutableArray<string> tags = default(System.Collections.Immutable.ImmutableArray<string>), Microsoft.CodeAnalysis.Completion.CompletionItemRules rules = null) -> Microsoft.CodeAnalysis.Completion.CompletionItem
```

Comment:
intentional.  The existing factory method was replaced with one with non-optional params (for binary compat).  A new factory method was added that allows for the new pieces of data to be passed along.  This is our pattern for adding new data to types without breaking binary compat.